.changeset/flat-melons-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/select': patch
+---
+
+popoverClassName prop added. The className is passed to the select options menu
```

Comment:
did we end up doing discovery around what popover pass through props we use/don't across components?